src/main/java/uk/gov/hmcts/divorce/common/notification/ApplicationIssuedNotification.java

```diff
@@ -0,0 +1,201 @@
+                email,
+                SOLE_APPLICANT_APPLICATION_ACCEPTED,
+                soleApplicant1TemplateVars(caseData, caseId),
+                languagePreference
+            );
+        } else {
+            log.info("Sending joint application issued notification to applicant 1 for case : {}", caseId);
+
+            notificationService.sendEmail(
+                email,
+                JOINT_APPLICATION_ACCEPTED,
+                commonTemplateVars(caseData, caseId, caseData.getApplicant1(), caseData.getApplicant2()),
+                languagePreference
+            );
+        }
+    }
+
+    @Override
+    public void sendToApplicant1Solicitor(final CaseData caseData, final Long caseId) {
+
+        final String email = caseData.getApplicant1().getSolicitor().getEmail();
+
+        if (caseData.getApplication().isSolicitorServiceMethod()) {
+            log.info("Sending Personal Service email to applicant solicitor.  Case ID: {}", caseId);
+
+            notificationService.sendEmail(
+                email,
+                APPLICANT_SOLICITOR_SERVICE,
+                templateVars(caseData, caseId),
+                ENGLISH
+            );
+        } else {
+            log.info("Sending Notice Of Proceedings email to applicant solicitor.  Case ID: {}", caseId);
+
+            notificationService.sendEmail(
+                email,
+                APPLICANT_SOLICITOR_NOTICE_OF_PROCEEDINGS,
+                solicitorNoticeOfProceedingsTemplateVars(caseData, caseId),
+                ENGLISH);
+        }
+    }
+
+    @Override
+    public void sendToApplicant2(final CaseData caseData, final Long caseId) {
+
+        final String email = caseData.getApplicant2EmailAddress();
+        final LanguagePreference languagePreference = caseData.getApplicant1().getLanguagePreference();
+
+        if (caseData.getApplicationType().isSole()) {
+            if (isNotBlank(email)) {
+                log.info("Sending sole application issued notification to respondent for case : {}", caseId);
+
+                notificationService.sendEmail(
+                    email,
+                    SOLE_RESPONDENT_APPLICATION_ACCEPTED,
+                    soleRespondentTemplateVars(caseData, caseId),
+                    languagePreference
+                );
+            }
+        } else {
+            log.info("Sending joint application issued notification to applicant 2 for case : {}", caseId);
+
+            notificationService.sendEmail(
+                email,
+                JOINT_APPLICATION_ACCEPTED,
+                commonTemplateVars(caseData, caseId, caseData.getApplicant2(), caseData.getApplicant1()),
+                languagePreference
+            );
+        }
+    }
+
+    @Override
+    public void sendToApplicant2Solicitor(final CaseData caseData, final Long caseId) {
+
+        log.info("Sending Notice Of Proceedings email to respondent solicitor.  Case ID: {}", caseId);
+
+        notificationService.sendEmail(
```

Comment:
this should also have a check for isSole

Reply:
but also, do we have the app2 solicitor email at this point? When the case is issued they won't have added their email

Reply:
I have added the isSole check and also a check for blank email.
https://github.com/hmcts/nfdiv-case-api/pull/1138/commits/07d3e0217c731630f46939f97e31a730dfffdc6e